packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -147,6 +153,22 @@ const onRenderParticipantDefault = (
             {callingParticipant.isMuted && (
               <Icon iconName="ParticipantItemMicOff" className={iconStyles} ariaLabel={strings.mutedIconLabel} />
             )}
+            {
+              /* @conditional-compile-remove(raise-hand) */ callingParticipant.raisedHand && (
+                <Stack horizontal={true} tokens={{ childrenGap: '0.2rem' }}>
+                  <Stack.Item>
+                    <Text>{callingParticipant.raisedHand?.raisedHandOrderPosition}</Text>
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Icon
```

Comment:
Color for this looks incorrect, should be the golden color:

| figma | testapp |
| -- | -- |
| ![image](https://github.com/Azure/communication-ui-library/assets/2684369/bae2ea18-417b-42f2-aed3-d6b950b77a06) | ![image](https://github.com/Azure/communication-ui-library/assets/2684369/1e2f9b67-4a6e-4332-87c3-55c1fbb37373) |